test/resources/activation_keys_test.rb

```diff
@@ -33,6 +33,10 @@ def test_create_and_update
 
       @name = "testakey1"
 
+      capture {
+        hammer.run(%W{activation-key delete --organization-label testcorp --name testakey1})
+      }
+
```

Comment:
I suggest using @name in the hammer.run call. (Probably can reduce scope to just name?)

The pattern I have been using is to clean up after the test (ie. assume testakey1 doesn't exist and then remove it after creation). Do you prefer this way for a reason?